File: Iniciante/1080.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner scanf = new Scanner(System.in);

        
        int n[] = new int[100], maior=0, i, pos=0;

        for(i=0; i < 100; i++){
            n[i] = scanf.nextInt();
        
            if(n[i] > maior){
                maior = n[i];
                pos = i+1;
            }
        }

        System.out.println(maior);
        System.out.println(pos);
    }
}

File: Iniciante/1080.c
#include <stdio.h>

int main(){

   int n[99], maior=0, i, pos;

    for(i=0; i < 100; i++){
        scanf("%d", &n[i]);
        
        if(n[i] > maior){
        maior = n[i];
        pos = i+1;
        }
    }

    printf("%d\n", maior);
    printf("%d\n", pos);

        
    return 0;
}